train.py

```diff
@@ -16,23 +16,23 @@
 
 @dataclass
 class Config:
-    lr_actor: float = field(default=0.0003)
-    lr_critic: float = field(default=0.0003)
+    lr_actor: float = field(default=3e-6)
+    lr_critic: float = field(default=3e-6)
```

Comment:
`3e-4` is the golden learning rate :)

Reply:
yea thats what i was thinking -- just messing around a bit